source/docs/api-reference/simulation/simulation-intro.rst

```diff
@@ -128,3 +128,30 @@ Some device ``SimState`` objects also contain outputs that can be used in simula
          // apply the new rotor position and velocity to the TalonFX
          m_talonFXSim.SetRawRotorPosition(m_motorSimModel.GetPosition());
          m_talonFXSim.SetRotorVelocity(m_motorSimModel.GetVelocity());
+
+High Fidelity CAN Bus Simulation
+--------------------------------
+
+All CTR-Electronics CAN devices support a high fidelity CAN bus, where the frames are simulated at a level similar to what the user sees on a real robot. This means that frames coming from a control frame will be sent at intervals corresponding to the control frame period, and frames from a device will be sent at the appropriate status frame period. In simulation, this may appear as a delay between setting a signal and getting its real value, or settings its real value and getting it in API.
```

Comment:
No dash between CTR and Electronics
"high fidelity CAN Bus simulation" instead of just "high fidelity CAN bus"
"what happens on a real robot" instead of "what the user sees on a real robot"
No comma after control frame period
No comma after In simulation and no comma after real value
"settings its real value" should be "setting its real value"